src/App.tsx

```diff
@@ -1,25 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [selectedPerPageValue, setSelectedPerPageValue] = useState<number>(5);
+  const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
if u set the default value to a simple type - typescript itself understands what type it is, so it is unnecessary.
For all such cases
```suggestion
  const [selectedPerPageValue, setSelectedPerPageValue] = useState(5);
  const [currentPage, setCurrentPage] = useState(1);
```